### DISCUSSION TITLE
建议增加方便的查看设备是否为本地连接的方式

### DISCUSSION THREAD

**Discussion by hi94740** [XiaoMi/ha_xiaomi_home#224] [Ideas]
现在好像找不到在哪能看到设备是否为本地连接

**Comment by Hugh696** [comment]
现在都不是本地，有仅针对Wi-Fi设备的本地模式，但暂时不建议使用。
等中枢固件更新之后就都是本地了

**Reply by hi94740** [reply]
> 现在都不是本地，有仅针对Wi-Fi设备的本地模式，但暂时不建议使用。
> 
> 等中枢固件更新之后就都是本地了

我没有中枢网关，想要确认wifi设备是不是本地连接的
